monkey/infection_monkey/utils.py

```diff
@@ -2,9 +2,13 @@
 import sys
 import shutil
 import struct
+import socket
 
 from infection_monkey.config import WormConfiguration
 
+LOCAL_IP = '127.0.0.1'
```

Comment:
Not correct. Local IP can be 127.0.0.1, IPv6, or localhost. Why do we need this?